package.json

```diff
@@ -19,6 +19,7 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/runtime": "^7.0.0",
     "@sentry/browser": "^4.4.2",
+    "algoliasearch": "^3.30.0",
```

Comment:
seems like a 5mo. old version, do we wanna use the latest (3.32.0)?

Reply:
Yeah sounds good